astroid/brain/brain_attrs.py

```diff
@@ -41,7 +40,7 @@ def attr_attributes_transform(node):
         if not isinstance(cdefbodynode, astroid.Assign):
             continue
         if isinstance(cdefbodynode.value, astroid.Call):
-            if cdefbodynode.value.func.as_string() != ATTR_IB:
+            if cdefbodynode.value.func.as_string() not in ATTRIB_NAMES:
```

Comment:
Maybe make `ATTRIB_NAMES` a frozenset instead of a tuple?

Reply:
Good point:
```
In [17]: timeit.timeit('"asdf" in s', 's = ("attr.s", "attrs", "attr.attrs", "attr.attributes")')                                             
Out[17]: 0.07539813104085624

In [18]: timeit.timeit('"asdf" in s', 's = frozenset(("attr.s", "attrs", "attr.attrs", "attr.attributes"))')                                  
Out[18]: 0.038608217029832304
```